### ISSUE TITLE
Brew unable to install older java versions

### ISSUE THREAD

**Issue by anirudhagar13** [sb2nov/mac-setup#230]
### I'm Submitting a ...

```
[x ] Bug report
[ ] Tool/language/etc documentation request
```

### Bug Location

brew would not be able to look for older versions of java, like java8 mentioned in the documentation, until cask is updated with versions.

### Other Information

Found the solutions at: 
- https://stackoverflow.com/questions/44632292/how-to-install-specific-java-version-using-homebrew

**Comment by apjanke** [comment]
I think a better approach is to use the `adoptopenjdk/openjdk` tap. That's the canonical source for JDKs straight from AdoptOpenJDK, and provides several versions and variants of the JDK.

https://stackoverflow.com/questions/53772023/how-to-install-java-8-on-osx-macos-mojave-with-homebrew/55775566

**Comment by simeg** [comment]
Fixed in #285.
